.ci/azure-pipelines.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       vmImage: 'ubuntu-latest'
 
     steps:
-      - script: 'echo "##vso[task.setvariable variable=JELLYFIN_VERSION]$(git describe --tags)"'
+      - script: echo "##vso[task.setvariable variable=JELLYFIN_VERSION]$( awk -F '/' '{ print $NF }' <<<'$(Build.SourceBranch)' | sed 's/^v//' )"
```

Comment:
`Build.SourceBranch` implies it's the name of a branch. For the apiclient (and Android repositories) we use tags to set the version and the branch can be anything (most of the time release branches though).

Reply:
Also, the gradle scripts are build with the `v` prefix in mind. I'm not sure if this change will break anything.

Reply:
Morning, sorry it was kind of late when I threw this together. Ill have a look at the line in a few.

However, are you implying that the build artifacts/assets created during the `gradlew publish` are using `*-vX.Y.Z*` in their names? (which was the sole reason I hastily created this PR, since that would also be an explanation why the files where never found)

Reply:
It should remove the `v` prefix for at least some parts the version is used but I'm not sure if it does that everywhere.

Reply:
> `Build.SourceBranch` implies it's the name of a branch. For the apiclient (and Android repositories) we use tags to set the version and the branch can be anything (most of the time release branches though).

Okay just had a quick look at the documentation for this, and seems to be sound logic.
The [documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#build-variables-devops-services) reads as follows: 
> Build.SourceBranch:
> The branch of the triggering repo the build was queued for. Some examples: 
> * .....
>* **When your pipeline is triggered by a tag: refs/tags/your-tag-name** 
> * ....

Reply:
remains the question: do the artifacts that get created by the gradlew task account for the `vX.Y.Z` version in their name or just `X.Y.Z`? (from my experience with Maven Java artifacts only have a X.Y.Z version in their name without the `v`)

Reply:
Okay, this may just be a shot in the dark, but with this code like this in the build.gradle.kts is it save to assume that the artifacts have no `v` in front of the version in their name:
```kts
// Versioning
allprojects {
	group = "org.jellyfin.apiclient"
	version = getProperty("jellyfin.version")?.removePrefix("v") ?: "SNAPSHOT"
}
```

Reply:
I looked into it and it should be fine 👍